vmaas/webapp/repos.py

```diff
@@ -105,6 +106,7 @@ def process_list(self, api_version, data):  # pylint: disable=unused-argument
 
         actual_page_size = 0
         repo_page_to_process, pagination_response = paginate(repos, page, page_size, filters=filters)
+        latest_repo_change = parse_datetime(self.cache.dbchange['last_change'])
```

Comment:
I think initializating to this value doesn't make much sense because `dbchange['last_change']` is the latest timestamp of all other values in dbchange. And it'll always be newer than any repo last_change.

Reply:
Let's initialize it to a first repo last_change for example. And the value can be null probably for a case when there is no repository in the output list.